docs/src/main/sphinx/release/release-440.md

```diff
@@ -0,0 +1,67 @@
+# Release 440 (7 Mar 2024)
+
+## General
+
+* Add [Snowflake connector](/connector/snowflake). ({issue}`17909`)
+* Add support for sub-queries inside `UNNEST` clauses. ({issue}`17953`)
+* Improve performance of {func}`arrays_overlap`. ({issue}`20900`)
+* Export JMX statistics for resource groups by default. This can be disabled
+  with the `jmxExport` resource group property. ({issue}`20810`)
+* Fix query failure when a `CHECK` constraint is null. ({issue}`20906`)
+* Fix query failure for aggregations over `CASE` expressions when the input
+  evaluation could throw an error. ({issue}`20652`)
+* Fix incorrect results when running a SQL routine with a single if/else
```

Comment:
Fix incorrect behavior of the `else` clause in a SQL routines with a single if/end condition.

Or 

Prevent ignoring else clause...